_layouts/donate-wna

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+---
+
+<div class="donate container">
+
+  <header class="page-header">
+    <h1 class="secondary-h1">{{ page.title }}</h1>
+    <div class="hr-h donate-hr"></div>
+
+    {% if page['Summary Text'] %}
+      <h4 class="news-summary">{{ page['Summary Text'] }}</h4> 
+    {% endif %}
+  </header>
+
+  <div class="donate-body">
+
+    <div class="donate-form">
+      {% include donate-form.html %}
```

Comment:
You will need to reference the wna hub donate form if you are using a separate form. otherwise donations will go to the same pool at HOT.

Reply:
Okay.